content/SDK/nodejs-sdk.mdx

```diff
@@ -39,8 +39,8 @@ import {
       set,
       schema,
       configs: {
-        'GREETING': 'hey',
-        'SUBJECT': 'configu node.js sdk'
+        GREETING: 'hey',
```

Comment:
this and the browser file should are auto generated from the README of the node and browser sdks
see `gatsby-node.js` - `onPreInit` - L:109